Translation/Extractor/Util/TranslationCollection.php

```diff
@@ -0,0 +1,65 @@
+<?php
```

Comment:
Missing license

Reply:
This repo has no license yet!

Reply:
Weird when we have license block in many files :(

Reply:
That is clearly a mistake. We only release the core in OSL.